bin/check_links.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+awesome_bot --skip-save-results --allow_ssl --allow-timeout --allow-dupe --allow-redirect --white-list https://github.com/$GITHUB_USER/istio *.md **/*.md
```

Comment:
sounds neat, can you add a comment on where this can be found, and maybe add it one of the circle ci check?